src/components/Pools/Pool/PoolV2.tsx

```diff
@@ -28,7 +28,7 @@ const PoolV2: React.FC<Props> = ({ type, setMenu, activeMenu, menuItems, miniChe
       case PoolType.superFarms:
         // return all farms with reward tokens address greater than 1 and with weight (multipler) greater than 0
         return (miniChefStakingInfo || []).filter(
-          (item) => (item?.rewardTokensAddress?.length || 0) > 1 && JSBI.greaterThan(item.multiplier, BIG_INT_ZERO),
+          (item) => (item?.rewardTokensAddress?.length || 0) > 0 && JSBI.greaterThan(item.multiplier, BIG_INT_ZERO),
```

Comment:
@prodesert22 subgraph tag is not coming

![Screenshot 2023-03-30 at 12 05 23 AM](https://user-images.githubusercontent.com/6604146/228635432-af73d1b1-e831-46f6-ac15-7b1bc6cfb829.png)